### PR TITLE
Updates ww gu

### PR DESCRIPTION
**Pull request by LianeHughes**
Wastewater methods for GU group have been updated, but their historic data will be shown. We have therefore maintained the 'old' graph scripts, with the suffix 'historic' and modified the existing scripts that feed into cron jobs

### CODE REVIEW

File: ClinMicro/lineage_four_recent.py [hunk 1]
```diff
@@ -271,6 +273,6 @@ def condition(x):
         ),
     ]
 )
-# fig.show()
+fig.show()
```
Comment (senthil10) [inline ClinMicro/lineage_four_recent.py:276]: comment ?

File: ClinMicro/lineage_one_plot.py [hunk 2]
```diff
@@ -222,7 +222,7 @@ def update_prop_graph(variants, lineage_groups):
             # ),
         ]
     )
-    # fig.show()
+    fig.show()
```
Comment (senthil10) [inline ClinMicro/lineage_one_plot.py:225]: comment?

**Review by senthil10 — COMMENTED** [review]
Looks good, just few forgotten re-commenting

File: ClinMicro/lineage_one_plot.py (hunk 2 above)
Comment (LianeHughes) [inline ClinMicro/lineage_one_plot.py:225]:
```suggestion
    # fig.show()
```

File: ClinMicro/lineage_four_recent.py (hunk 1 above)
Comment (LianeHughes) [inline ClinMicro/lineage_four_recent.py:276]:
```suggestion
# fig.show()
```

File: wastewater/gothenburg_covid_historic.py [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+#         )
+#     ]
+# )
+# Below can show figure locally in tests
+# fig.show()
+
+# # Prints as a json file
+# fig.write_json("wastewater_gothenburg.json")
+
+fig.write_image("wastewater_gothenburg_line_historic.png")
```
Comment (senthil10) [inline wastewater/gothenburg_covid_historic.py:139]: Also noticed its writing png here, but I assume this is just a backup script so this doesn't matter?

**Pull request merged**